.gitignore

```diff
@@ -18,6 +18,10 @@ dist/
 build/
 lib/
 coverage/
+target
+test/rust/
+metaplex-tests-ledger
+solana-validator.yml
```

Comment:
This is also no longer needed as I pointed out previously ... config is written to tmp dir and cleaned up by amman.